initdb.d/01_main.sql

```diff
@@ -27,7 +27,8 @@ CREATE TABLE datasets (
     id                  SERIAL PRIMARY KEY,
     stable_id           TEXT UNIQUE,
     title               TEXT,
-    description         TEXT
+    description         TEXT,
```

Comment:
where would the `description` info come from ?

Reply:
dunno, but we added it when we made the v2 schema ^^